soccer/WindowEvaluator.h

```diff
@@ -0,0 +1,75 @@
+//
```

Comment:
I also think pragma once looks slightly clean than the c style ifndefs.
Is there a way to change clion default templates lol?

Probs some documentation for this file would be good.